README.md

```diff
@@ -174,7 +174,7 @@ The received responses may originate in one of two types of service:
 1. **Transcription**: responses generated by a Speech Recognition service which converts speech in the input media stream to text. Responses with "transcription" service type will have the same language of the input media stream (as specified in the order).
 2. **Translation**: responses generated by a Machine Translation service which translates "transcription" responses (in the input language) to one or more target languages (as specified in the order). 
 
-_Note: words in responses from the "translation" service type are assigned timings which are heuristically distributed across the time-spans of their counterpart words in the source language, as these timings may be useful for some purposes (e.g. displaying translated captions in sync with the media)._ 
+_Note: phrasings and word order in translated responses may be significantly different from those of the source language, due to the nature of translation to another language. Since the translated words were never really uttered, they do not have "real" timings. However, words in "translation" responses are assigned timings which are heuristically distributed within the time boundaries of the source language utterance, as these timings may be useful for some purposes (e.g. displaying translated captions in sync with the media)._ 
```

Comment:
```suggestion
_Note: Due to natural differences between languages, translated responses may diverge in word count and word order. Since translated words were never really uttered in the original audio, they do not have "real" timings. Therefore, words in translation responses are assigned timings which are heuristically distributed within the time boundaries of the source language utterance. Heuristic timings may be used for synchronization purposes like displaying translated content in alignment with the media._ 
```

Reply:
Accepted suggestion